uint8 inst_num
Data data_t
---
Data data_r